.github/workflows/ci.yml

```diff
@@ -24,16 +24,17 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
-      - name: Run Lacci tests
-        run: CI_RUN='true' bundle exec rake lacci_test
+      # Currently failing
+      # - name: Run Lacci tests
+      #   run: CI_RUN='true' bundle exec rake lacci_test
```

Comment:
Hm. This turns off all Lacci tests?

Reply:
I filed an issue when I found that I was getting the same Laci failures on CI on this branch, locally on this branch, and locally on main.

Maybe this was too aggressive, but thought that could be the best approach to isolate that issue in a separate PR

Maybe it's not reproducible though? I think you have commented on this issue so let's revisit

I definitely do want these tests. But if they were going red on main I wanted to toggle them until resolved